packages/core/src/actor/connection.ts

```diff
@@ -96,6 +99,10 @@ export class Conn<S, CP, CS, V, I, AD, DB extends AnyDatabaseProvider> {
 		return this.__persist.t;
 	}
 
+	public get isConnected(): boolean {
```

Comment:
this property name is confusing

i recommend we exppose a `get status(): Status`

where `type Status = "connected" | "reconnecting"`